### DISCUSSION TITLE
AGX Color Profile

### DISCUSSION THREAD

**Discussion by Leonobu** [EatTheFuture/image_tools#15] [Questions & Troubleshooting] [answered]
For some reason, with Blender 4.0 I lost all the Blender default profiles because of my custom OCIO color. Does anyone knows how to solve this?

**Comment by cessen** [comment, marked as answer]
I think this is a misunderstanding of how OCIO works.  OCIO (including in Blender) only uses a single configuration at a time.

So when you generate an OCIO configuration with ETF OCIO Maker, and then point Blender at that configuration, Blender *completely ignores* the default configuration that comes bundled with Blender, and *only* uses the OCIO-Maker-created configuration you pointed Blender at.

The reason Filmic, etc. are still there with the configurations from OCIO Maker is because I took great pains in OCIO Maker to exactly recreate the configuration from Blender 3.x, except with the additional spaces the user specifies.  I haven't yet had the time to do that with Blender 4.x's default OCIO configuration, which includes AgX.  Although I am working on it.
